docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+Welcome and thank you for considering contributing to `${REPO_NAME}`.  We're delighted to have you as part of our community.
+
+Following these guidelines will help us effectively manage the contribution process for eveyone involved in the contribution process. It communicates that you respect the time of the developers managing and developing this open source project. In return, we will reciprocate that respect by addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+# Contributing
+
+We have open sourced `${REPO_NAME}` to help make it easy for you to use LUSID and welcome contributions from our community. There are many ways to contribute, from writing code samples, improving the documentation, submitting bug reports and feature requests to writing code which can be incorporated into `${REPO_NAME}` itself. This section guides you through submitting a contribution.
+
+## Reporting bugs
+
+**If you find a security vulnerability, please do NOT open an issue. Any security issues should be submitted directly to soc@finbourne.com.**
+
+If you think you have found a bug, first make sure you are testing against the latest version of `${REPO_NAME}` as your issue may have already been fixed. Also search all open and closed issues to see if your issue has already been answered.
+
+If you are unable to find your issue then please raise an issue using the provided template. Try to provide as much information as possible that will help us reproduce the problem, provide the ticket that _you_ would like to receive if you had to fix the issue.
+
+## Submitting changes
+
+If you have a change that you would like to contribute, please find or open an issue about it first as there may be somebody already working on it, or there may be particular issues that you should know about before implementing the change.
+
+1. Fork the repository to your own GitHub account, this [GitHub guide](https://help.github.com/en/articles/fork-a-repo) describes how to do this.
+1. Clone the project to your local machine
+1. Make your changes and ensure that all the tests successfully pass.
+1. Update your local repository and rebase your changes on top of the latest `master` branch. We prefer that your changes are squashed into a single commit.
+1. Push you changes to your fork
+1. Submit a [pull request](https://help.github.com/en/articles/about-pull-requests) to the main repository
+
+## Code review process
+
+The Core Team reviews pull requests on a regular basis and will give feedback on the corresponding issue in the repo. After feedback has been given it is expected that you respond within two weeks, the pull request may be closed if it does not show any signs of activity. An automated build will run as part of your pull request, at a minimum this needs to be passing before your contribution is accepted. 
+
+# Getting in touch
+
+Please don't use the issue tracker for support questions, for help on using LUSID please contact support@lusid.com
```

Comment:
do we really want to add this to every repository as a default.  Appreciate it's completely needed for the SDK's, but other OSS projects may not even be related.  either way it has to be added individually, or removed individually, and I'm not wedded to either.

Reply:
I had the same though and was waiting to see if anyone had a view, I've removed